build/npm/import-language-xlfs.js

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+let path = require("path");
+let gulp = require('gulp');
+let minimist = require('minimist');
+
+function update(options) {
+	let location = options._;
+
+	gulp.src(`${location}/drop/loc/**/*.xlf`)
```

Comment:
If this drop folder doesn't exist it would be helpful to print an error message informing the user what steps they need to do first